### PR TITLE
Feature/record fps

### PR DESCRIPTION
**Pull request by tam-nguyen-senseye**
* added package: `react-native-ffmpeg`

* updated `Video.getMetadata()` to extract and compile metadata from a local video file

* refactored `Video.getMetadata()` and `Task.getMetadata()` to be async

* refactored `Video.upload()` by removing `uri` param

* updated Android example app's minSdk to 24

* updated Usage examples in README

* code cleanup

[ch18406]

### CODE REVIEW

File: src/api/models/Task.ts [hunk 1]
```diff
@@ -99,13 +99,15 @@ export default class Task {
...
+   * @returns A `Promise` that will produce the compiled {@link metadata}.
    */
-  public getMetadata() {
+  public async getMetadata() {
     if (this.videos.length > 0) {
-      let videoData = this.videos[0].getMetadata();
+      let videoData = await this.videos[0].getMetadata();
       videoData = { ...videoData.info, ...videoData };
-      delete videoData.info;
+      videoData.info = undefined;
```
Comment (Haplo-Dragon) [inline src/api/models/Task.ts:110]: Is there a reason we're setting `.info` to `undefined` when we flatten the data here, but deleting it when we flatten things on line 150 in `Session.uploadJsonData`? The two methods seem similar to me, but maybe I'm missing something?

**Review by Haplo-Dragon — APPROVED** [review]
Nice - the README update is very helpful, too!

File: src/api/models/Task.ts (hunk 1 above)
Comment (tam-nguyen-senseye) [reply src/api/models/Task.ts:110]: good question, idk ... i think that might've been the `yarn lint --fix` 🤔

File: README.md [hunk 2]
```diff
@@ -50,15 +50,62 @@ For local development, one of the dependencies is `babel-plugin-module-resolver`
...
+          if (job.result !== undefined) {
+            console.log('result: ' + JSON.stringify(job.result));
+          }
+        }
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, 3000);
+  ```
```
Comment (ach206) [inline README.md]: 👍🏾 👍🏾

File: example/src/screens/FullDemoScreen.tsx [hunk 3]
```diff
@@ -61,18 +61,18 @@ export default function FullDemoScreen() {
       // upload complete
       clearInterval(uploadPollId);
 
-      let video_urls: string[] = [];
-      values[0].forEach((v: any) => {
-        video_urls.push(v.s3_url);
+      let videoUrls: string[] = [];
```
Comment (ach206) [inline example/src/screens/FullDemoScreen.tsx:64]: 👀  is that camelCase I see? lol

**Review by ach206 — APPROVED** [review]

**Review by bobbys-dev — APPROVED** [review]
Able to see fps recorded in metadata 👍

**Pull request merged**